### ISSUE TITLE
Debugger crashes when entering separate file in debug mode

### ISSUE THREAD

**Issue by boorac** [trufflesuite/truffle#826]
- [* ] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

`truffle debug <txn>` enters the debugger however it crashes when entering a new file.

## Steps to Reproduce

Try debugging a transaction in which the debugger has to step through multiple files.

## Expected Behavior

Debugger should continue normally.

## Actual Results

`AFXCrowdsale.sol:` is the entry point, `AFXToken.sol` is a separate file.
```
AFXCrowdsale.sol:

79:   function () external payable {
80:     uint256 tokensThatWillBeMintedAfterPurchase = msg.value.mul(rate);
81:     if ((stage == CrowdsaleStage.PreICO) && (token.totalSupply() + tokensThatWillBeMintedAfterPurchase > totalTokensForSaleDuringPreICO)) {
                                                 ^^^^^^^^^^^^^^^^^^^                                                                           

debug(development:0xc6ead1d6...)> 

AFXToken.sol:

TypeError: Cannot read property 'line' of undefined

```
## Environment

* Operating System: `Ubuntu 16.04`
* Ethereum client: `geth 1.8.1-stable`
* Truffle version:
 ```
Truffle v4.1.0 (core: 4.1.0)
Solidity v0.4.19 (solc-js)

```
* node version:`v8.9.1`
* npm version: `5.5.1`

# Workaround
Use a flattener, compile and debug from a single large file:

`truffle-flattener contracts/crowdsale/AFXCrowdsale.sol | pbcopy` etc.

**Comment by boorac** [comment]
@cgewecke AFXCrowdsale does not inherit from AFXToken or vice versa. I'm able to debug inherited code when using a conglomerate file. I'm not sure if it has to do with inheritance but it definitely has to do with separate files.

**Comment by cgewecke** [comment]
@boorac Thanks yes, clearly. Thanks for the work-around too, nice.

**Comment by boorac** [comment]
Happy to contribute :)

**Comment by eduardonunesp** [comment]
In my case the issue happens when I tried to call a function in a token, for instance that test token inherits from `StandardToken` from `OpenZeppelin`, then I called the `transfer` on `truffle console`. After that I executed `truffle debug <transaction hash>`. The debugger worked, but when try to jump in the `transfer` function the same error described above happen.

**Comment by gnidan** [comment]
I think this should be fixed when trufflesuite/truffle-debugger#39 goes out in the next release!

Closing for cleanup reasons. Thanks!
